packages/apollo/tests/subscriptions/graphql-ws.spec.ts

```diff
@@ -37,7 +36,12 @@ describe('graphql-ws protocol', () => {
           },
           subscriptions: {
             'graphql-ws': {
-              onConnect: (context: Context<any>) => {
+              onConnect: (
+                context: Context<
+                  any,
+                  { socket: { close: (number, string) => any | boolean } }
+                >,
+              ) => {
```

Comment:
Did this to fix the typescript errors, if anyone can point me towards the right type that is used here, I'm happy to adjust this line.